packages/editor/src/panels/scenes/SceneItem.tsx

```diff
@@ -26,37 +26,33 @@ import { PopoverState } from '@ir-engine/client-core/src/common/services/Popover
 import { deleteScene } from '@ir-engine/client-core/src/world/SceneAPI'
 import { StaticResourceType } from '@ir-engine/common/src/schema.type.module'
 import { timeAgo } from '@ir-engine/common/src/utils/datetime-sql'
-import { useClickOutside } from '@ir-engine/common/src/utils/useClickOutside'
-import { useHookstate } from '@ir-engine/hyperflux'
-import RenameSceneModal from '@ir-engine/ui/src/components/editor/panels/Scenes/modals/RenameScene'
+import RenameSceneModal from '@ir-engine/editor/src/panels/scenes/RenameSceneModal'
 import ConfirmDialog from '@ir-engine/ui/src/components/tailwind/ConfirmDialog'
+import { Popup } from '@ir-engine/ui/src/components/tailwind/Popup'
 import Button from '@ir-engine/ui/src/primitives/tailwind/Button'
 import Text from '@ir-engine/ui/src/primitives/tailwind/Text'
 import Tooltip from '@ir-engine/ui/src/primitives/tailwind/Tooltip'
-import { default as React, useEffect, useRef } from 'react'
+import { default as React } from 'react'
 import { useTranslation } from 'react-i18next'
 import { BsThreeDotsVertical } from 'react-icons/bs'
 import { LuTrash } from 'react-icons/lu'
 import { MdOutlineEdit } from 'react-icons/md'
-import { twMerge } from 'tailwind-merge'
 
 type SceneItemProps = {
   scene: StaticResourceType
-  moveMenuUp?: boolean
   handleOpenScene: () => void
   refetchProjectsData: () => void
   onRenameScene?: (newName: string) => void
   onDeleteScene?: (scene: StaticResourceType) => void
 }
 
-export const SceneItem = ({
+export default function SceneItem({
```

Comment:
Please create a PR for MT, as same control is used in it. And this change is breaking the import in MT.

Reply:
yes had made the change in [the MT PR](https://github.com/theinfinitereality/irpro-multitenancy/pull/460/files)